build.gradle.kts

```
@@ -1,6 +1,6 @@
 buildscript {
     dependencies {
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.30")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.30")
```

Might as well bump all the way to latest
```suggestion
        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.31")
```

---

I purposefully kept it low because Jetpack Compose only officially support 1.5.30

---

But I guess bumping it here would only affect the library dependency which should be fine

---

Jetpack Compose does now support 1.5.31